File: Wacomi.API/Models/BlogFeed.cs
using System;

namespace Wacomi.API.Models
{
    public class BlogFeed
    {
        public int Id{ get; set;}
        public int BlogId { get; set;}
        public Blog Blog{get; set;}
        public DateTime? PublishingDate { get; set;}
        public string Title{ get; set;}
        public string Url{ get; set;}
        public string ImageUrl{ get; set;}
    }
}

File: Wacomi.SPA/src/app/_models/BlogFeed.ts
import { BlogFeedComment } from "./BlogFeedComment";
import { ShortComment } from "./ShortComment";

export interface BlogFeed {
    id: number;
    blogId: number;
    blogTitle: string;
    publishingDate: Date;
    title: string;
    url: string;
    imageUrl: string;
    blogImageUrl: string;
    writerName: string;
    ownerId: number;
    likedCount? : number;
    commentCount? : number;
    isLiked : boolean;
    
    //Client side only
    displayComments?: boolean;
    shortComments?: ShortComment[];
}
